### PR TITLE
Chlp

### PR DESCRIPTION
**Pull request by TheLady**

### CODE REVIEW

**Comment by nodatall** [comment]
Please wait to merge your PR until after it has been reviewed and you have addressed changes.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
+## Team Name - lazy-puffin
+Candida and Davina
+
 This repo contains resources, challenges and plans for learning the basics of Javascripts.
 
 If you're working on a Solo goal, checkout the [Solo Study Guide](solo_study.md)
 
 If you're working on a team project, checkout the [Team Practice Guide](team_practice.md)
+
+License:
```
Comment (nodatall) [inline README.md:12]: [License: MIT](https://opensource.org/licenses/MIT)

File: exercism/hamming/hamming.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+var Hamming = function() {}
+
+Hamming.prototype.compute = function (aString, bString) {
+  console.log('we are running the compute function')
```
Comment (nodatall) [inline exercism/hamming/hamming.js:4]: Don't leave `console.log`s in your PRs.

File: exercism/hamming/hamming.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  if(aString == bString) {
+    return 0
+  }
+
+
+
```
Comment (nodatall) [inline exercism/hamming/hamming.js:10]: remove extra whitespace

**Comment by TheLady** [comment]
Thanks - should we create another matching one for review?

File: exercism/hamming/hamming.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+var Hamming = function() {}
+
+Hamming.prototype.compute = function (aString, bString) {
+  console.log('we are running the compute function')
+  if(aString == bString) {
```
Comment (nodatall) [inline exercism/hamming/hamming.js:5]: I prefer spaces between if and parameters: `if (aString == bString)`

**Comment by TheLady** [comment]
Thank you!

**Pull request merged**